spec/requests/registrations_spec.rb

```diff
@@ -1,28 +1,46 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/BlockLength
 RSpec.describe 'Registrations', type: :request do
-
-  shared_examples "sucessful password update" do
-    let(:update_params) { { password: new_password, password_confirmation: new_password } }
+  shared_examples 'successfully updates user and redirects' do
     before do
       sign_in user
       put user_registration_path, params: update_params
+      user.reload
     end
 
     it { expect(response).to redirect_to(edit_user_registration_path) }
+    it { expect(user.comment_notify).to be(false) }
   end
 
-  describe "update password" do
-    let(:new_password) { "foobar_w!th_@" }
+  describe '#update' do
```

Comment:
Since the test flow goes from the controller method calling until the redirection